OpenXmlPowerTools/WorksheetAccessor.cs

```diff
@@ -571,11 +571,13 @@ public static PivotTablePart CreatePivotTable(SpreadsheetDocument document, stri
                     {
                         value = GetCellValue(document, sourceSheet, column, row);
                         if (value is double)
+                        {
```

Comment:
If the value is double, then you don't need to convert. You can use `if(value is double dbl)` and then have access to it as a double. The Convert is if it is *not* already a double. The original logic doesn't really make sense, either